docs/hadoop_streaming_hue.md

```diff
@@ -0,0 +1,131 @@
+Big data within [Hadoop](http://hadoop.apache.org/) clusters is mainly analyzed by means of the [MapReduce](https://en.wikipedia.org/wiki/MapReduce) paradigm. Usually, specific Java jobs are created for this purpose, but sometimes different programming languages may want to be used, such as Python or even Shell scripting. In that cases, [Hadoop Streaming](https://hadoop.apache.org/docs/current/hadoop-streaming/HadoopStreaming.html) is the feature the IoT Platform integrators need.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-iot-stack/pull/22/commits/74218e91b5cd5a93dd7a1898a2a7d0f77d36713c